### DISCUSSION TITLE
When I run Media Pipe on Android, FPS is too low. Can I know how to optimize it?

### DISCUSSION THREAD

**Discussion by KIhoonNam** [endink/Mediapipe4u-plugin#158] [Q&A] [answered]
I'm developing an Android version of MediaPipe like the title, but the FPS is too low. Is there a way to raise the FPS?

60 FPS is maintained and when MediaPipe is activated, it drops to 17 FPS.

**Comment by endink** [comment, marked as answer]
Sorry, This version of google mediapipe for holistic is very slow on Android, and no way to improve it , but google recently upgraded the new holistic task API that looks like performance improved, but I hasn't tried it yet. 

There's  a lot things on my hands recently and have no time to update the M4U.

Sorry for that.

**Reply by doncaleb020** [reply]
@KIhoonNam - Have you taken build for Android ? Can we know the target platform and it's out comes
